src/db.rs

```diff
@@ -85,8 +83,73 @@ pub fn insert_connection(connection: Connection) -> Result<Uuid> {
     Ok(nwuuid)
 }
 
+pub fn insert_policy(policy: &policy::Policy) -> Result<u64> {
+    let mut dbconn = get_dbconn()?;
+
+    let allowed_digests_json = serde_json::to_string(&policy.allowed_digests)?;
+    let allowed_policy_json = serde_json::to_string(&policy.allowed_policies)?;
+    let allowed_build_ids_json = serde_json::to_string(&policy.allowed_build_ids)?;
+
+    let mysqlstr = format!(
+        "INSERT INTO policy (allowed_digests, allowed_policies, min_fw_api_major, min_fw_api_minor,
+                               allowed_build_ids, create_date, valid)
+                               VALUES({:?}, {:?}, {}, {}, {:?}, NOW(), 1)",
+        allowed_digests_json,
+        allowed_policy_json,
+        policy.min_fw_api_major,
+        policy.min_fw_api_minor,
+        allowed_build_ids_json
+    );
+    dbconn.query::<u64, String>(mysqlstr)?;
+    Ok(dbconn.last_insert_id())
+}
+
+pub fn get_policy(pid: u64) -> Result<policy::Policy> {
+    let mut dbconn = get_dbconn()?;
+
+    let pidstr = pid.to_string();
```

Comment:
no need to convert to string (see for example in `delete_policy`).

Reply:
Fixed this by removing pidstr line and using pid directly.